deploy/charts/trust-manager/values.yaml

```diff
@@ -101,6 +101,10 @@ app:
     # -- If false, disables the default seccomp profile, which might be required to run on certain platforms
     seccompProfileEnabled: true
 
+# -- List of secrets trust-manager is allowed to read/write. Used when the secret target is set when creating
+# a bundle. If the list is empty it means read/write access to all secrets.
+authorizedSecrets: ["ca-bundle"]
```

Comment:
suggestion: I'm not sure if this is the safest way to do this.

I have a few concerns:

- Having a default here feels wrong - since most clusters won't need to be able to read/write to secrets called `ca-bundle`. I'd rather the default be that there's nothing extra which we have access to, so that users have to _explicitly_ expand trust-manager's permissions.
- Having an empty list confer permissions to _everything_ is really confusing to me (the intuitive reading would be that an empty list means there are no allowed secrets at all)
- There doesn't seem to be an obvious way to allow no secrets, in the event that I don't want to use secrets at all.

I'd personally rather the default be "no secrets are authorized". How about something like the following?

```yaml
secretTargets:
    # -- If set to true, enable writing trust bundles to Kubernetes Secrets as a target
    enabled: false

    # -- A list of secret names which trust-manager will be permitted to read and write across all namespaces. These will be the only allowable Secrets that can be used as targets. Must be a non-empty list  if secretTargets.enabled is true
    authorizedSecrets: [""]
```

Reply:
What would be the difference between `authorizedSecrets: [""]` and `authorizedSecrets: []` ?

Reply:
Whoops, that was a typo, my bad! I think the default should be `authorizedSecrets: []`.

If enabled is set to true, we should check that authorizedSecrets is a list with at least one valid secret name. `""` wouldn't be a valid secret name.

Reply:
okay, I add another option authorizedSecretsAll to indicate permission to all. And now empty list of authorizedSecrets means access to nothing